### ISSUE TITLE
Biggoron misleadingly says to come back after a few days when handing in Eyedrops

### ISSUE THREAD

**Issue by ETR-BTF** [TestRunnerSRL/OoT-Randomizer#1956]
It's misleading both because you do actually need the Claim Check to get that item (i.e. it's not given "for free"), and because the 3 day waiting period has been patched out. The best solution would probably be to have him say something along the lines of "Come back with the Claim Check for another item" because that also works if the trade quest items are not shuffled.

![image](https://user-images.githubusercontent.com/63906538/231794273-00b16734-ffc0-401d-bfdd-715f8383f2e7.png)

**Comment by r0bd0g** [comment]
The three-day waiting period being patched out is misleading trade quest shuffle or no.

(Setting to have that be an optional speedup when???)

**Comment by ETR-BTF** [comment]
Yeah I guess that part of the issue title was superfluous lol.

**Comment by fenhl** [comment]
Should be an easy fix. Any suggestions on the worrrrrding?

**Comment by fenhl** [comment]
Maybe the best way to handle it would be to skip the claim check step entirely when it's not shuffled and instead just give the BGS check (but still replace eyedrops with it in the inventory). Then we can make the wording of this text box specific to shuffled claim check.

**Comment by ETR-BTF** [comment]
> Should be an easy fix. Any suggestions on the worrrrrding?

"Brrrring me the Claim Check...
to rrreceive anotherrrrrr item..."

Or something like that lol. Swap "item" for "gift" if you think that word is misleading (because "item" is sometimes used in the community to denote things that aren't junk or keys)

**Comment by r0bd0g** [comment]
I don't know that the vanilla text needs updating here just b/c there's a lot of texts that don't make sense when items are all shuffled around. But this case is a little different, at least in the case of trade shuffle off, because we really did remove the 3-day wait. Except I mean he's not wrong? 0 is a number of days, and you can wait longer if you want. If you must insist on updating the text I would reword it to remove reference to waiting, but not care about whether the new text makes sense in the context of trade shuffle.
